### PR TITLE
Restructure configuration between "development and docker" environment

### PR DESCRIPTION
**Pull request by butterngo**
Hi @thangchung I was done with organizing "Configuration", now we can deal environment between dev and docker

### CODE REVIEW

File: samples/AggregationRestApi/appsettings.Development.json [hunk 1]
```diff
@@ -3,15 +3,15 @@
     "GrpcMappers": [
       {
         "GrpcMethod": "/Greet.Greeter/SayHello",
-        "GrpcHost": "127.0.0.1:5003"
+        "GrpcHost": "localhost:5003"
       },
       {
         "GrpcMethod": "/ProductCatalog.Product/GetProducts",
-        "GrpcHost": "127.0.0.1:5002"
+        "GrpcHost": "localhost:5003"
```
Comment (thangchung) [inline samples/AggregationRestApi/appsettings.Development.json]: I think this is 5002, right?

File: samples/AggregationRestApi/appsettings.Development.json [hunk 2]
```diff
@@ -3,15 +3,15 @@
...
       },
       {
         "GrpcMethod": "/ProductCatalog.Product/CreateProduct",
-        "GrpcHost": "127.0.0.1:5002"
+        "GrpcHost": "localhost:5003"
```
Comment (thangchung) [inline samples/AggregationRestApi/appsettings.Development.json]: The same with above

File: samples/OcelotGateway/Properties/launchSettings.json [hunk 3]
```diff
@@ -1,12 +1,19 @@
 {
+  "iisSettings": {
```
Comment (thangchung) [inline samples/OcelotGateway/Properties/launchSettings.json]: I don't think we need to add IIS settings in here

File: samples/OcelotGateway/ocelot.Development.json [hunk 4]
```diff
@@ -7,8 +7,8 @@
       "DownstreamScheme": "http",
       "DownstreamHostAndPorts": [
         {
-          "Host": "aggregator",
-          "Port": 80
+          "Host": "localhost",
+          "Port": 5001
```
Comment (thangchung) [inline samples/OcelotGateway/ocelot.Development.json]: we can point it direct to the real gRPC service in the downstream

File: samples/OcelotGateway/Properties/launchSettings.json (hunk 3 above)
Comment (butterngo) [reply samples/OcelotGateway/Properties/launchSettings.json]: @thangchung  your mean remove it

**Comment by thangchung** [comment]
@butterngo Thank you. I'm happy to merge it

**Pull request merged**